DFC.Digital/DFC.Digital.Web.Sitefinity.Core/SitefinityExtensions/CompUIConfigSettingService.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.ServiceModel;
+using System.ServiceModel.Activation;
+
+namespace DFC.Digital.Web.Sitefinity.Core
+{
+    /// <summary>
+    /// Sitefinity web service.
+    /// </summary>
+    [ServiceBehavior(IncludeExceptionDetailInFaults = true, InstanceContextMode = InstanceContextMode.Single, ConcurrencyMode = ConcurrencyMode.Single)]
+    [AspNetCompatibilityRequirements(RequirementsMode = AspNetCompatibilityRequirementsMode.Required)]
+    public class CompUIConfigSettingService : ICompUIConfigSettingService
```

Comment:
Think we already have a service that can get appSettings, can we not use that? Also doesnt have any security around this, means open for anyone to get any appSettings??!!.

Reply:
This was resolved now and added the changes into Sitefinity backend services instead of public one